app/interactors/content/update_interactor.rb

```diff
@@ -58,7 +58,7 @@ def update_preview
   end
 
   def change_note_params
-    params.require(:revision).permit(:update_type, :change_note)
+    params.permit(:update_type, :change_note)
```

Comment:
This will raise a warning about the other parameters. You'd see something like "Unpermitted parameters: :title, :summary, :contents". May want to change this to something like `{ update_type: params[:update_type], change_note: params[:change_note] }`

Just a thought too - it doesn't look we do anything to stop these params going in when we're on the first edition so we might want this to have a `return {} if edition.first?`

Reply:
Oh yes, sorry - you did mention. `#slice` and `#to_h` are also protected (with an exception, it turns out). I've also pushed an extra commit to add that new check in 👍.